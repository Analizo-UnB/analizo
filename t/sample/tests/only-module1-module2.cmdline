egypt module1.c.131r.expand module2.c.131r.expand
