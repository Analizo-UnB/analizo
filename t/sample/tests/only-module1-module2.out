digraph callgraph {
"main" -> "say_bye" [style=solid];
"main" -> "say_hello" [style=solid];
}
